lua/terrortown/entities/roles/none/shared.lua

```diff
@@ -0,0 +1,19 @@
+ROLE.Base = "ttt_role_base"
+
+ROLE.index = ROLE_NONE
+
+---
+-- @ignore
+function ROLE:PreInitialize()
+	self.color = COLOR_CUSTOMGRAY
+
+	self.abbr = "none"
+
+	self.defaultTeam = TEAM_NONE
+	self.defaultEquipment = {}
+
+	self.builtin = true
+	self.scoreKillsMultiplier = 0
+	self.scoreTeamKillsMultiplier = 0
```

Comment:
these have changed since you forked TTT2. Either remove them completely or take a look at the base